js/pages/settings.js

```diff
@@ -29,7 +29,17 @@ let canWeVoicemailHome;
 let canWeVoicemailOther;
 let userData;
 let template = '';
-
+let middleNameRow;
+let suffixRow;
+let preferredFirstNameRow;
+let mobilePhoneRow;
+let mobilePhoneVoicemailRow;
```

Comment:
what do we gain from declaring all of these here?

Reply:
These were added when I realized any phone number can be null - a user can enter any of three phone numbers at sign up. Only one is required for the form to validate, so we can't assume any of those fields will exist for any given user. Adding them here allows for visibility toggling based on user actions.

Reply:
Will it be better to hold these values in an object (eg `let rows={middleNameRow:null, surfixRow:null, ...otherRows};`)?

Reply:
Just personal preferences. Not real changes needed.